docs/configure_your_operations.md

```diff
@@ -234,22 +242,22 @@ class Book implements ResourceInterface
 
 It will configure this route for your `show` operation.
 
-| Name            | Method | Path        |
-|-----------------|--------|-------------|
-| app_book_show   | GET    | /books/{id} |    
+| Name          | Method | Path        |
```

Comment:
`Show` operation will look better right after `Index` operation instead of the last place after `Delete`

Reply:
fixed